visual-python/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM python:3.12-slim-bookworm
+FROM python:3.11-slim-bookworm
```

Comment:
Reduce from 3.12 -> 3.11 since 3.12 is not officially supported by robotframework-seleniumlibrary: https://github.com/robotframework/SeleniumLibrary/blob/master/setup.py#L44